k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
I'm still new to Docker... do we need a public registry at all?  Can't we simply build a container locally and run it locally?